cmd/edge-orchestration/main.go

```diff
@@ -191,5 +195,8 @@ func orchestrationInit() error {
 		}
 	}
 
+	//Creating MQTT client object to initiate cloudsync
+	cloudsyncmgr.StartMQTTClient(mqttBrokerURL, mqttport)
```

Comment:
u mean u want the api for cloud sync to be written under orchestrationapi package?

Reply:
No. The orchestrationapi package orchestrates controllers like discovery, scoring, storage, etc. Please see [Build](https://github.com/lf-edge/edge-home-orchestration-go/blob/9c738811ac353669c73a33b5f6a5d0d330090c7a/internal/orchestrationapi/orchestration.go#L175) and [Start](https://github.com/lf-edge/edge-home-orchestration-go/blob/9c738811ac353669c73a33b5f6a5d0d330090c7a/internal/orchestrationapi/orchestration.go#L201) function of orchestrationapi. 
The common mqtt should be implemented under `internal/common/mqtt` and cloudsync feature should be implemented under `internal/controller/cloudsyncmgr`.

Reply:
the mqqt is put under common/mqtt and the api for the cloudsync would be under cloudsyncmgr. But the APi part i will push in separate commit so for now i have moved the connection portion under mqtt for reusability

Reply:
Same mqtt can act as client for local mqtt connection between devices. Cloudsyncmgr would have send API for now to send data to cloud which can be later extended to retrieve data from cloud(Pull).
This PR has only the mqtt connect part and publish was planned to be added as separate PR followed by API(cloudsyncmgr) in another PR. When cloudsyncmgr is implemented it would be done as scoring, storage and so.

Reply:
Done!